### DISCUSSION TITLE
User can change status

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#1494] [Ideas]
Hi,
I noticed in the latest update 5.11.0.5094, that it is possible to tell the users that cannot change status mode. This is a nice feature!
There are some bugs I found, for example, if you are setting your gender to anything other than Male, then the TeamTalk client automaticly gives you errors when you connect to the server.
I was wondering if this feature can still be put in the latest version, because it's a good feature to have, for those who do not want their users to put a status message on the server. Also it's good for audio players, so that if you don't want the audio player to show what is playing it would be blocked from being able to display it with TeamTalk.
I hope that this feature will still be added for future versions of TeamTalk, it's a great idea and great feature.
Thanks for reading!

**Comment by bear101** [comment]
The feature was removed due to old clients will always change status due to gender.

**Reply by computertech1012** [reply]
Hi @bear101,
The problem is not with the older clients, for example, if I used TeamTalk 5.2.3 the male gender was already set automaticly and it did not give errors. It happens if your gender is not set to male only.
I really hope that you can fix it and add this feature back, because this is a good feature, just needs a bit of work.
I think that status should not count against gender but if you test with an example client, 5.2.3, you will understand. It only happens if the gender is not set to male. And since 5.2.3 sets to gender 1 by default it does not give the error.
In the latest version, if you set gender to Neutral then that's when you will encounter the error.
Let me know if I should provide more information.
